### DISCUSSION TITLE
question about upscale images

### DISCUSSION THREAD

**Discussion by PouyaFakhari** [lllyasviel/Fooocus#2915] [Q&A]
Fooocus tool is really great (Although its ability to improve quality is relatively good), but I am looking for a photo enhancement tool ai like this to increase the quality and details of the photo (like the magnific ai tool, which is expensive, unfortunately). Do you have a suggestion? Anyway, thank you for your amazing tool.

**Comment by Sammicen** [comment]
Thanks for sharing! Pictures in your collection or downloaded from the Internet are most likely to appear pixelated, blurry, or blocky. I have used an AI repair tool before, and the effect is quite good. If you are interested, you can learn about the latest AI technology von VanceAI. 
1. [Image Upscaler](https://vanceai.com/de/image-enlarger/)
2. [Image Sharpener](https://vanceai.com/de/sharpen-ai/)
3. [Image Enhancer](https://vanceai.com/de/image-enhancer/)
